sources/config_reader.c

```diff
@@ -1203,7 +1209,8 @@ od_config_reader_ldap_storage_credentials(od_config_reader_t *reader,
 static int od_config_reader_rule_settings(od_config_reader_t *reader,
 					  od_rule_t *rule,
 					  od_extention_t *extentions,
-					  od_storage_watchdog_t *watchdog)
+					  od_storage_watchdog_t *watchdog,
+					  od_group_t *group)
```

Comment:
why you gives group as argument? you have already had it in "od_rule_t *rule"

Reply:
thanks, fixed